bgpd/bgp_updgrp_adv.c

```diff
@@ -486,9 +486,10 @@ void bgp_adj_out_set_subgroup(struct bgp_dest *dest,
 	 * the route wasn't changed actually.
 	 * Do not suppress BGP UPDATES for route-refresh.
 	 */
-	if (CHECK_FLAG(bgp->flags, BGP_FLAG_SUPPRESS_DUPLICATES)
-	    && !CHECK_FLAG(subgrp->sflags, SUBGRP_STATUS_FORCE_UPDATES)
-	    && adj->attr_hash == attr_hash) {
+	if (CHECK_FLAG(bgp->flags, BGP_FLAG_SUPPRESS_DUPLICATES) &&
+	    !CHECK_FLAG(dest->flags, BGP_NODE_LABEL_CHANGED) &&
```

Comment:
This sure seems like a *Different* bug fix than what is actually going on in this commit.  This needs to be broken out into it's own commit imo.

Reply:
> What about `bgp labeled-unicast explicit-null`? Is this cleaned up?

yes

Reply:
@ton31337 -> Too me this change looks like we need to look at the BGP_PATH_LINK_BW_CHG value too here, right?

Reply:
> @ton31337 -> Too me this change looks like we need to look at the BGP_PATH_LINK_BW_CHG value too here, right? Do we send this data to our peers?

👍, but not with this PR?

Reply:
oh agreed.  I just was looking at the code and that flag seemed like a hole in sending updates to our peers when a link bandwidth changes for whatever reason, but everything else stays the same.